Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,108 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = (
+      %w[brew core cask] +
+      OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") } +
+      OFFICIAL_CASK_TAPS
+    ).freeze
+
+  sig { returns(CLI::Parser) }
+  def contributions_args
+    Homebrew::CLI::Parser.new do
+      usage_banner "`contributions` [<email>]"
+      description <<~EOS
+        Contributions to Homebrew repos for a user.
+      EOS
+
+      flag "--from=",
+           description: "Date (ISO-8601 format) to start searching contributions."
+
+      flag "--to=",
+           description: "Date (ISO-8601 format) to stop searching contributions."
+
+      comma_array "--repos=",
```

Comment:
```suggestion
      comma_array "--repositories=", "--repos",
```

Reply:
`comma_array` doesn't take multiple `name`s. Yet. I suppose I could fix that.